.storybook/themes/commerce.scss

```diff
@@ -334,6 +334,46 @@
 					max-width: var(--content-scale-width),
 					width: 100%,
 				),
+				carousel-indicator-dots-container: (
+					display: flex,
+					gap: var(--global-spacing-4),
+					padding: var(--global-spacing-4) 0 0 0,
+					justify-content: center,
+					align-items: center,
+				),
+				carousel-indicator-dot: (
+					height: var(--global-spacing-3),
+					width: var(--global-spacing-3),
+					border: var(--global-border-width-1) solid var(--border-color),
+					border-radius: var(--border-radius-circle),
+					background-color: transparent,
+				),
+				carousel-indicator-dot-active: (
+					border: none,
+					height: var(--global-spacing-4),
+					width: var(--global-spacing-4),
+					background-color: var(--global-neutral-4),
+				),
+				carousel-indicator-thumbnails-container: (
+					padding: var(--global-spacing-4) var(--global-spacing-3) 0 var(--global-spacing-3),
+					width: 100%,
+					overflow-x: scroll,
+					overflow-y: hidden,
+					height: var(--global-spacing-11),
+					white-space: nowrap,
+				),
+				carousel-indicator-thumbnail: (
+					margin: 0 var(--global-spacing-3),
+					height: var(--global-spacing-7),
+					width: var(--global-spacing-7),
+					padding: var(--global-spacing-1),
+					background-color: transparent,
+					opacity: 40%,
+				),
+				carousel-indicator-thumbnail-active: (
+					border: var(--global-border-width-1) solid var(--global-black),
+					opacity: 75%,
```

Comment:
There is no `opacity` in the designs for the active thumbnail which would be this value would be `100%`

Reply:
oh huh. it must've changed since yesterday

<img width="860" alt="Screen Shot 2022-09-13 at 10 53 17" src="https://user-images.githubusercontent.com/5950956/189948796-8a6e5922-1fd2-44e7-8e4a-80a4422416c8.png">

Reply:
oh I see, that other design was for the icon not the picture

Reply:
updated -- thanks